### DISCUSSION TITLE
Curation with variable width or height

### DISCUSSION THREAD

**Discussion by justijndepover** [awcodes/filament-curator#338] [Q&A]
Hi

Excuse me if this is in the docs, but I cant seem to find it.

Is it possible to define a curation where either the width or the height is variable?
I would like to crop the images to a width of 1500px but keep the aspect ratio of the original image.

This is for a gallery plugin where I show all the thumbnails, but when opening an image, I would like to show the original aspect ratio, but still a cropped version.

Thanks

**Comment by howdu** [comment]
You can set the height/weight to 'auto'. 
```

<x-curator-glider
    :media="$image"
    :srcset="['1680w', '1024w', '640w']"
    height="auto"
```

https://github.com/awcodes/filament-curator/pull/194
